pom.xml

```diff
@@ -48,7 +48,15 @@
 
 	<build>
 		<plugins>
-			<plugin>
+            <plugin>
```

Comment:
What is this supposed to achieve? If you want to export the package just give it a `@Version` annotation, the rest is handled by the configuration inherited from the parent pom.